corehq/apps/reminders/signals.py

```diff
@@ -1,23 +1,20 @@
 from casexml.apps.case.models import CommCareCase
 from casexml.apps.case.signals import case_post_save
-from corehq.apps.reminders.models import (CaseReminderHandler,
-    REMINDER_TYPE_DEFAULT)
 from corehq.apps.reminders.tasks import case_changed
 from dimagi.utils.logging import notify_exception
 
+
 def case_changed_receiver(sender, case, **kwargs):
     """
-    Fires reminders associated with the case, if any exist.
+    Spawns a task to update reminder instances tied to the given case.
     """
     try:
-        handler_ids = CaseReminderHandler.get_handler_ids(case.domain,
-            reminder_type_filter=REMINDER_TYPE_DEFAULT)
-        if len(handler_ids) > 0:
-            case_changed.delay(case._id, handler_ids)
+        case_changed.delay(case.domain, case._id)
```

Comment:
(same comment about spawning a task for each case save, still seems better than doing the processing in the receiver signal and most of the time should be a virtual no-op)